origin-discovery/src/listener/handler_identity.js

```diff
@@ -1,9 +1,10 @@
 const Web3 = require('web3')
 const logger = require('./logger')
 
+const _bridgeModels = require('origin-bridge')
```

Comment:
Should this be `origin-bridge/src/models`?

Reply:
doh ! I Fixed.